openapi-integration-tests/src/test/java/io/ballerina/openapi/idl/client/IDLClientGenPluginTests.java

```diff
@@ -119,4 +123,18 @@ public void testWithLocalPathJSON() throws IOException, InterruptedException {
         Assert.assertNotNull(matchingFiles);
         Assert.assertEquals(matchingFiles.length, 1);
     }
+
+    @Test
+    public void testWithClientClassName() throws IOException, InterruptedException {
+        String classContent = "public isolated client class Client {";
+        Process process = executeRun(DISTRIBUTION_FILE_NAME, TEST_RESOURCE.resolve("project_15"), new LinkedList<>());
+        process.waitFor();
+        Path clientFile =
+                Path.of(RESOURCE.resolve("client-idl-projects/project_15/generated/bar/client.bal").toString());
+        Stream<String> expectedServiceLines = Files.lines(clientFile);
+        String generatedContent = expectedServiceLines.collect(Collectors.joining(System.lineSeparator()));
+        expectedServiceLines.close();
+        Assert.assertTrue((generatedContent.trim()).replaceAll("\\s+", "")
+                .contains(classContent.trim().replaceAll("\\s+", "")));
```

Comment:
Can't we just do 
```suggestion
        Assert.assertTrue(generatedContent.contains(expectedClientSignature));
```

in here (where `String clientSignature = "public isolated client class Client {";`)?

Reply:
We suppose to remove the whitespace because most of the time we encountered a test failure due to the different encoding values to the whitespace. Therefore we decided to remove all the whitespace when doing some content comparison. :)

Reply:
Are you saying that it's intermittent? and if it's not, shall simplify the check and see if it work?